update.sh

```diff
@@ -43,19 +57,32 @@ for version in "${versions[@]}"; do
 	md5="${fullVersion##* }"
 	fullVersion="${fullVersion% $md5}"
 
+	if [ "$version" != '7' ]; then
+		md5='composer'
+	fi
 	echo "$version: $fullVersion ($md5)"
 
-	for variant in fpm-alpine fpm apache; do
+	for variant in {apache,fpm}-buster fpm-alpine3.12; do
+		[ -e "$version/$variant" ] || continue
 		dist='debian'
-		if [[ "$variant" = *alpine ]]; then
+		if [[ "$variant" = *alpine* ]]; then
 			dist='alpine'
 		fi
 
-		sed -r \
-			-e 's/%%PHP_VERSION%%/'"${phpVersions[$version]:-$defaultPhpVersion}"'/' \
-			-e 's/%%VARIANT%%/'"$variant"'/' \
-			-e 's/%%VERSION%%/'"$fullVersion"'/' \
-			-e 's/%%MD5%%/'"$md5"'/' \
-		"./Dockerfile-$dist.template" > "$version/$variant/Dockerfile"
+		phpImage="${phpVersions[$version]:-$defaultPhpVersion}-$variant"
+		if [ "$version" = '7' ]; then
+			# 7 has no release in drupal/recommended-project
+			# so its Dockerfile is based on the old template
+			sed -ri \
+				-e 's/^(FROM php:).*$/\1'"${phpImage}"'/' \
+				-e 's/^(ENV DRUPAL_VERSION ).*$/\1'"$fullVersion"'/' \
+				-e 's/^(ENV DRUPAL_MD5 ).*$/\1'"$md5"'/' \
+			"$version/$variant/Dockerfile"
```

Comment:
This is going to be "fun" if 7 ends up needing structural changes (much like Python 2 was before we finally bit the bullet and made a second set of templates :sweat_smile:).

(I think it's fine for now though.)

Reply:
It seems that now Drupal 8 is going to EOL before 7 (https://www.drupal.org/psa-2020-06-24), so yeah I'll make some templates. :cry:

Reply:
Drupal 7 templates added.